src/config.rs

```diff
@@ -2487,6 +2523,9 @@ pub struct TiKvConfig {
     #[online_config(skip)]
     pub log: LogConfig,
 
+    #[online_config(skip)]
```

Comment:
Should be configurable.

Reply:
you mean supporting online config?

Reply:
Yep

Reply:
agree but maybe this will be another PR, let's make this small and easy to merge (exceeded 1k lines)